packages/autocomplete-plugin-redirect-url/src/templates.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+export const defaultTemplates = {
+  item({ createElement, state }) {
+    return (
+      <div className="aa-ItemWrapper">
+        <div className="aa-ItemContent">
+          <div className="aa-ItemContentBody">
+            <div className="aa-ItemContentTitle">
+              {state.query}
+            </div>
+          </div>
+        </div>
+        <div className="aa-ItemActions">
+          <button className="aa-ItemActionButton">
```

Comment:
does it change anything if this button is removed?

Reply:
Good call. I had assumed assumed the styling should be kept for consistency with the other plugins, but the core stuff (color, hover effect, size) is still available without. Just removed it.